src/core/constructs/cloud_output.py

```diff
@@ -160,7 +160,8 @@ def evaluate_dynamic_output(
                 new_rv = getattr(intermediate_value, func_name)(*xargs)
             elif (not xargs) and (not kwargs):
                 new_rv = getattr(intermediate_value, func_name)()
-
+            else:
```

Comment:
These lines are redundant since `pass` is just an empty execution statement.

Reply:
It is, but we should raise an exception if none of the previous if-elif matches